go.mod

```diff
@@ -241,7 +248,5 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-replace github.com/pingcap/tidb/parser => ./parser
```

Comment:
Please keep parser a separated package. It is intended to be a standalone component, since it may be imported by others.